doc/user/data/self_managed/latest_versions.yml

```diff
@@ -2,6 +2,6 @@ terraform_gcp_version: v0.5.7
 terraform_azure_version: v0.5.7
 terraform_aws_version: v0.5.7
 terraform_helm_version: v0.1.28
-operator_helm_chart_version: v25.2.13
-orchestratord_version: v0.162.0
+operator_helm_chart_version: v25.2.14
```

Comment:
Should the other values be updated too?
https://github.com/MaterializeInc/materialize/commit/356d1dbc3536d2f45519e64889d21643bd47b82b